src/activities/Elsa.Activities.Http/Middleware/HttpEndpointMiddleware.cs

```diff
@@ -172,5 +145,62 @@ public class HttpEndpointMiddleware
                 }
             }
         }
+
+        private async Task<bool> AuthorizeAsync(
+            HttpContext httpContext,
+            IActivityBlueprintWrapper<HttpEndpoint> httpEndpoint,
+            IWorkflowBlueprint workflowBlueprint,
+            CollectedWorkflow pendingWorkflow,
+            IHttpEndpointAuthorizationHandler authorizationHandler,
+            CancellationToken cancellationToken)
+        {
+            var authorize = await httpEndpoint.EvaluatePropertyValueAsync(x => x.Authorize, cancellationToken);
+
+            if (!authorize)
+                return true;
```

Comment:
This property is a boolean within an instance of a HttpEndpointActivity to signify whether it requires the request to be authorized.
So if you do not need to be authenticated, then this method returns true.
The code if correct for its purpose.
If the variable was named "authorized" then it would have been questionable.